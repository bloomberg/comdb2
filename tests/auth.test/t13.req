# Change to OP user
set user 'user1'
set password 'password1'

# Create a no-OP user
put password 'tunable' for 'tunable'

set user 'tunable'
set password 'tunable'
select "no-OP sets tunables"
put genid48 enable
put genid48 disable
put rowlocks enable
put rowlocks disable
put schemachange convertsleep 10
put schemachange commitsleep 10
put tunable sc_async 1
put tunable sc_async 0
