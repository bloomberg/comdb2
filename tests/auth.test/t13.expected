("no-OP sets tunables"='no-OP sets tunables')
[put genid48 enable] failed with rc -3 User does not have OP credentials
[put genid48 disable] failed with rc -3 User does not have OP credentials
[put rowlocks enable] failed with rc -3 User does not have OP credentials
[put rowlocks disable] failed with rc -3 User does not have OP credentials
[put schemachange convertsleep 10] failed with rc -3 User does not have OP credentials
[put schemachange commitsleep 10] failed with rc -3 User does not have OP credentials
[put tunable sc_async 1] failed with rc -3 User does not have OP credentials
[put tunable sc_async 0] failed with rc -3 User does not have OP credentials
